KC7-Foundation/kc7_scoreboard
# Restructured and optimized user_profile.html for better readability and structure

### Description
This pull request introduces a new JavaScript and CSS file for the `user_profile.html` page. Instead of having a `<script>` tag and CSS code within the file itself, this PR aims to improve coding practices and readability by separating them into different files.

### Changes Made
- Added `user_profile.js`.
- Added `user_profile.css`.
- Removed the redundant `<div class="card-body">` element within the `table-responsive` container.
- Eliminated unnecessary inline styles for image width and popover content.

### How to Test
1. Check out this branch.
2. Navigate to `.../profile/_jibba_`.
3. Verify that no styling changes have been made from the original version. This PR primarily focuses on structural changes.
